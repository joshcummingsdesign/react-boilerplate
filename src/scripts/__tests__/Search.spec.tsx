import * as React from 'react';
import { shallow } from 'enzyme';
import Search from '../components/Search';

test('Search renders correctly', () => {
  const component = shallow(<Search />);
  expect(component).toMatchSnapshot();
});
