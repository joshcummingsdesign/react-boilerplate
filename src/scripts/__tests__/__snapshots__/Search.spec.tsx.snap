// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders correctly 1`] = `
<div
  className="search"
>
  <h1>
    Search for a Post
  </h1>
  <form
    onSubmit={[Function]}
  >
    <input
      placeholder="Search"
      type="text"
    />
  </form>
</div>
`;
